.rubocop.yml

```diff
@@ -0,0 +1,122 @@
+# use the shared Yast defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+Metrics/AbcSize:
+  Max: 129
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/AbcSize:
+  Max: 30
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/BlockNesting:
+  Max: 5
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/BlockNesting:
+  Max: 2
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 385
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 250 # TODO this should be lower for new code
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/ModuleLength:
+  Max: 385
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/ModuleLength:
+  Max: 100
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/CyclomaticComplexity:
+  Max: 29
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/CyclomaticComplexity:
+  Max: 13 # TODO this should be lower for new code
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+# Configuration parameters: AllowURI, URISchemes.
+Metrics/LineLength:
+  Max: 299
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+# Configuration parameters: AllowURI, URISchemes.
+Metrics/LineLength:
+  Max: 100 # TODO this should be lower for new code
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 160
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 30 # TODO this should be lower for new code
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/PerceivedComplexity:
+  Max: 34
+  Exclude:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+Metrics/PerceivedComplexity:
+  Max: 14 # TODO this should be lower for new code
+  Include:
+    - 'src/lib/**/*.rb' # be more strict for new code in lib
+
+# be more strict then rubocop, 5 is too much
+Metrics/ParameterLists:
+  Max: 4
+
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+Style/AndOr:
+  EnforcedStyle: conditionals
+
+Style/Documentation:
+  Include:
+    - 'src/lib/**/*.rb' # force well documented code only for new code
+
+Style/FileName:
+  Include:
+    - 'src/lib/**/*.rb' # force correct names on new code
+
+Style/HashSyntax:
+  Include:
+    - 'src/lib/**/*.rb' # force new hash style only for new code
+
+# JR: I found lambda easier to read and understand then ->()
```

Comment:
Yeah, i just looked for some yast module that have it updated so i copy it from yast-bootloader module :wink: